students/mia/users/views.py

```diff
@@ -34,3 +34,21 @@ def post(self, request):
 
         except KeyError:
             return JsonResponse({"message":"Key_Error"}, status=400)
+    
+class SignInView(View):
+    def post(self,request):
+        try:
+            input_data = json.loads(request.body)
+
+            # non-existent log-in info 
+            if not User.objects.filter(email = input_data['email']).exists():
+                return JsonResponse({"message":"INVALID_USER"}, status=400)
+            # wrong password
+            if User.objects.get(email = input_data['email']).password != input_data['password']:
+                return JsonResponse({"message":"INVALID_USER"}, status=401)
```

Comment:
```suggestion
            if not User.objects.filter(email = email, password = password).exists():
                return JsonResponse({'Message': 'SUCCESS'}, status=200)       
            return JsonResponse({'Message': 'INVALID_USER'}, status=401)
```

이렇게 한번에 해도 되지 않을까요~?

Reply:
네! ㅎㅎ 말씀 하신대로 반영했습니다~ 감사해용